layouts/shortcodes/observability_pipelines/prerequisites/datadog_agent.md

```diff
@@ -1,7 +1,7 @@
 You already have the Datadog Agent installed to collect and route your logs to [Datadog Log Management][2001]. If you do not have Datadog Agents set up, see the [Datadog Agent documentation][2002] for more information.
 
 You have the following information available:
-- The Datadog Agent address, including the port, that your Observability Pipelines Worker will listen on for logs.
+- The Datadog Agent address, including the port. The Observability Pipelines Worker listens on this bind address to receive logs from your Datadog Agents.
```

Comment:
```suggestion
- The "Datadog Agent address": the socket address that the Observability Pipelines Worker will be listening for to receive logs from the Datadog Agent.
```

Trying to use similar language to what we have in-app, WDYT?

![Screenshot 2024-06-10 at 2 24 57 PM](https://github.com/DataDog/documentation/assets/133862/af9f8681-1569-42bb-be68-e68e656a43a4)

Reply:
DM'd on this. We rephrased it and won't use the wording in the UI because it doesn't read well.